engine/src/main/java/org/terasology/rendering/opengl/OpenGLMesh.java

```diff
@@ -187,7 +188,7 @@ public int addToBatch(Transform transform, Transform normalTransform, TFloatList
         int n = 0;
         int c = 0;
         for (int v = 0; v < data.getVertices().size(); v += VERTEX_SIZE) {
-            javax.vecmath.Vector3f vert = new javax.vecmath.Vector3f(data.getVertices().get(v), data.getVertices().get(v + 1), data.getVertices().get(v + 2));
+            Vector3f vert = new Vector3f(data.getVertices().get(v), data.getVertices().get(v + 1), data.getVertices().get(v + 2));
```

Comment:
Please wrap data.getVertices() and data.getNormals() (below) in variables, to shorten the lines and improve readability.

Reply:
Bigger but potentially out of scope problem - the `addToBatch` method is not used anywhere (as reported by IntelliJ). Would removing it altogether be a better idea? If so, I'd rather not make the change here since the method is going to be removed anyway.

Reply:
Ok, let's remove it then. Let's see if anything breaks.